numpy/_core/_add_newdocs.py

```diff
@@ -3636,6 +3636,19 @@
     >>> x.item((2, 2))
     1
 
+    For an array with object dtype, elements are returned as-is.
+
+    >>> a = np.array([np.int64(1)], dtype=object)
+    >>> a.item() #return np.int64
+    np.int64(1)
+    >>> a.item().item() #return python scalar
+    1
+    >>> a = np.array([np.longdouble(1.11)], dtype=object)
+    >>> a.item() #return np.longdouble
+    np.longdouble('1.1111111')
+    >>> a.item().item() #still return np.longdouble
+    np.longdouble('1.1111111')
+
```

Comment:
Let's remove these parts of the example - my feeling that this is too in-the-weeds for a docstring example and is generally not of interest to typical users (the folks from the relevant discussion in the original issue are downstream library developers).

Reply:
Done.